### PR TITLE
PP-11

### PR DESCRIPTION
**Pull request by sanjay-techuz**
add raised by, resolved by and action cell in risks table also remove action cell from project list

### CODE REVIEW

**Comment by sanjay-techuz** [comment]
[add Raised By and Resolved By field in risks](https://trello.com/c/op0QS13B/11-add-raised-by-and-resolved-by-field-in-risks)

File: src/components/Table/Project/Risks/index.jsx [hunk 1]
```diff
@@ -41,21 +74,75 @@ export default function Index(props) {
     setRowsPerPage(+event.target.value);
     setPage(0);
   };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log('Submitted');
```
Comment (ketanp-techuz) [inline src/components/Table/Project/Risks/index.jsx:80]: Remove the console

File: src/pages/Projects/ProjectDetail/View/DailyStanup.jsx [hunk 2]
```diff
@@ -113,7 +106,6 @@ export default function DailyStanup(props) {
   console.log(data);
```
Comment (ketanp-techuz) [inline src/pages/Projects/ProjectDetail/View/DailyStanup.jsx:106]: Remove it

File: src/pages/GlobalDailyStandups/GlobalDailyStandups.jsx [hunk 3]
```diff
@@ -9,112 +9,94 @@ import TimelineDot from '@mui/lab/TimelineDot';
...
+        createdBy: 'Ramina hadid ',
+        project: 'Ice cream sandwich',
+        description: 'Because you need strength',
+        date: '09:30',
+      },
+      {
+        createdBy: 'Jessica doe',
+        project: 'Cupcake',
+        description: 'Because you need strength',
+        date: '09:30',
```
Comment (ketanp-techuz) [inline src/pages/GlobalDailyStandups/GlobalDailyStandups.jsx:81]: It's value will be date or time? change accordingly

**Review by ketanp-techuz — APPROVED** [review]

**Pull request merged**